src/main/java/com/redhat/devtools/lsp4ij/usages/LSPUsageSearcher.java

```diff
@@ -89,9 +91,14 @@ public void processElementUsages(@NotNull PsiElement element, @NotNull Processor
                     }
                 }
             }
+        } catch (ProcessCanceledException pce) {

```

Comment:
I changed this behavior in one place based on an observed error. This same change may need to be rolled out anywhere there is `try ... catch (Exception) ...` where the caught exception could be `ProcessCanceledException`. That should only be caught/handled in VERY specific situations. Otherwise it should be propagated to the IDE.

Reply:
It should be done like this with the main classes.